src/addon/file_server/directory_entry.rs

```diff
@@ -11,8 +12,10 @@ pub struct DirectoryEntry {
     pub(crate) size: String,
     pub(crate) len: u64,
     pub(crate) entry_path: String,
-    pub(crate) created_at: String,
-    pub(crate) updated_at: String,
+    pub(crate) date_created_string: String,
+    pub(crate) date_modified_string: String,
+    pub(crate) date_created_exact: DateTime<Local>,
+    pub(crate) date_modified_exact: DateTime<Local>,
```

Comment:
Why not just use `DateTime<Local>`?

```suggestion
    pub(crate) date_created: DateTime<Local>,
    pub(crate) date_modified: DateTime<Local>,
```

Reply:
Problem is, the defaut formatting of DateTime is very long and messy

Reply:
The value that comes here is _virtual_ used as a "DTO" to then render the formatted value, is not intended for the user to read it until it gets to the presentational layer (Handlebars) where it is formatted with a human friendly text.

Reply:
So how can you render it better by default?

Reply:
> The value that comes here is _virtual_ used as a "DTO" to then render the formatted value, is not intended for the user to read it until it gets to the presentational layer (Handlebars) where it is formatted with a human friendly text.

By default, it's not rendered human-friendly at all

Reply:
You must use the format method from chrono

Reply:
That's already how it's implemented

Reply:
And have you tried using helpers? https://github.com/sunng87/handlebars-rust/blob/master/examples/render.rs

Otherwise lets have a single `String` for `created_at` and another for `updated_at`

Reply:
Here is the documentation: https://docs.rs/handlebars/latest/handlebars/struct.Helper.html Helpers run on render-time, so it should be possible to use them on the `DateTime` instance so we can format the value as desired

Reply:
> Otherwise lets have a single String for created_at and another for updated_at

The problem with this is, you can't sort dates given human-readable strings